javascript/knockout/households/index.js

```diff
@@ -0,0 +1,72 @@
+// your code goes here ...
+function HouseholdMember (age, relationship, smokes) {
+  const member = this
+
+  member.age = age
+  member.relationship = relationship
+  member.smokes = smokes
+
+  member.asListItem = ko.computed(function () {
+    const smokingIcon = '🚬'
+    const noSmokingIcon = '🚭'
+    const smokesIcon = member.smokes ? smokingIcon : noSmokingIcon
+    return `${member.relationship}: ${member.age} ${smokesIcon}`
+  })
+}
+
+function HouseholdsViewModel () {
+  const model = this
```

Comment:
The convention here is to use `self` or `vm` for view model. This isn't really the model. That would be the household or member. This is the view model.